### PR TITLE
Add validation groups for body converter on doc

### PR DESCRIPTION
**Pull request by Soullivaneuh**
As suggested by @lsmith77 on https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1200#issuecomment-156798753.

### CODE REVIEW

File: Resources/doc/request_body_converter_listener.rst [hunk 1]
```diff
@@ -91,5 +91,23 @@ The validation errors will be set on the ``validationErrors`` controller argumen
         // ...
     }
 
+You can configure the validation groups used by the validator
+via the ``validator`` option:
+
+.. code-block:: php
+
+    /**
+     * @ParamConverter("post", converter="fos_rest.request_body", options={"validator"={"groups"={"foo", "bar"})
```
Comment (daFish) [inline Resources/doc/request_body_converter_listener.rst]: There are 2 `}` missing at the end before the closing `)`.

**Comment by Ener-Getick** [comment]
:+1:

File: Resources/doc/request_body_converter_listener.rst (hunk 1 above)
Comment (Soullivaneuh) [inline Resources/doc/request_body_converter_listener.rst]: That's @Ener-Getick fault! :-P

Will correct it.

**Comment by Soullivaneuh** [comment]
Review corrected.

File: Resources/doc/request_body_converter_listener.rst (hunk 1 above)
Comment (Ener-Getick) [inline Resources/doc/request_body_converter_listener.rst]: Not the same fault ! https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1200#issuecomment-156137323 :D

**Comment by Ener-Getick** [comment]
Thank you @Soullivaneuh

**Pull request merged**